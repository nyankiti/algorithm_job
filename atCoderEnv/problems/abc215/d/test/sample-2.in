65 100000
26132 46933 49471 42836 56764 97034 60628 80721 89762 50282 29902 71693 79201 2423 20789 51945 63950 14857 16533 53487 3371 65989 84759 96626 16832 9965 20826 98183 52256 23549 92473 24417 49742 69417 68692 10226 25037 23628 26810 30873 36107 4782 83842 10660 91463 10938 22613 7371 90457 46747 24837 37265 23799 97508 8637 17738 22969 73634 52763 21931 44589 85148 52713 86134 12234
