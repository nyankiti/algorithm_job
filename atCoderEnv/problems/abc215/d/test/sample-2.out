12237
1
43
